consensus.go

```diff
@@ -293,7 +318,10 @@ func (p *Pbft) setRound(round uint64) {
 func (p *Pbft) runAcceptState(ctx context.Context) { // start new round
 	_, span := p.tracer.Start(ctx, "AcceptState")
 	defer span.End()
-
+	startTime := time.Now()
```

Comment:
Why do not put this at the beginning of `runCycle`? You would only need to do it once per all the states.